auth_session_timeout/models/ir_config_parameter.py

```diff
@@ -14,23 +14,34 @@ class IrConfigParameter(models.Model):
     _inherit = 'ir.config_parameter'
 
     @tools.ormcache(skiparg=0)
-    def get_session_parameters(self, db):
+    def _auth_timeout_get_parameter_delay(self, db):
         param_model = self.pool['ir.config_parameter']
         cr = self.pool.cursor()
```

Comment:
Sounds like you're going for a real fix, I ❤️  it and thank you! Let's do like this in both of the cursor areas:

```
with openerp.registry(self.env.cr.dbname).cursor() as cr:
    ...operations...
```

This will kill the try/finally.

Now, I do have a question regarding this & forgive if it's basic - I don't know this module. Why are we opening new cursors in these spots?